source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -492,9 +494,9 @@ function add_handbook_templates( $templates ) {
  * @return string
  */
 function filter_code_content( $content ) {
-	$post = get_post();
+	$post_type = get_post_type();
 
-	if ( ! is_single() || ! is_parsed_post_type( $post->post_type ) ) {
```

Comment:
These changes should probably not be done in this branch.

Reply:
Originally I changed it because I used an extra `$post_type` below (not anymore for now) and I saw this PHP warning locally somehow:
<img width="1718" alt="image" src="https://user-images.githubusercontent.com/18050944/218599570-4b099964-ff30-4896-91ca-aa0a46ecf722.png">
After applying this change, the error was gone. Not sure if you saw the warning as well on your side.

Reply:
I think it's the right change but we should make it in trunk and rebase.